### ISSUE TITLE
Component > Button > #a11y contrast issue

### ISSUE THREAD

**Issue by arbydev** [carbon-design-system/carbon#337]
On the Component > Button page
http://carbondesignsystem.com/components/button/code

The hover and focus color of the button color does not meet AA contrast ratios. This makes it a non-highly visible focus indicator.

**Comment by tw15egan** [comment]
FYI, hover does not need to pass AA standards, as long as the focus state has a non-color indicator (we add a border around our buttons). 

[For more information](https://webaim.org/blog/wcag-2-0-and-link-colors/)
